pkgs/by-name/nu/numcpp/package.nix

```diff
@@ -9,15 +9,19 @@
 }:
 stdenv.mkDerivation (finalAttrs: {
   pname = "numcpp";
-  version = "2.13.0";
+  version = "2.14.1";
 
   src = fetchFromGitHub {
     owner = "dpilger26";
     repo = "NumCpp";
-    rev = "Version_${finalAttrs.version}";
-    hash = "sha256-+2xd8GNMSKPz801lfMAcHIkmidKd+xM8YblkdFj3HZk=";
+    tag = "Version_${finalAttrs.version}";
+    hash = "sha256-8ghHTRneLeiFgzF9zHFjFu46qopc3l2Svku1H06XQTw=";
   };
 
+  postPatch = ''
+    sed -i '1i cmake_policy(SET CMP0004 OLD)' test/pytest/src/CMakeLists.txt
```

Comment:
I think this will fail once we update to CMake 4 ([doc](https://cmake.org/cmake/help/latest/policy/CMP0004.html)). Unfortunately it is not obvious to me how the issue could be fixed properly.